### PR TITLE
Even More Docs Style Updates

### PR DESCRIPTION
**Pull request by braunjj**
## Summary & Motivation
More style updates 
- Cleaned up the top nav
- Denser code sample text
- New table of contents style
- Cleaner code highlighting
- MermaidJS tweaks (more to come)
- Cleaner breadcrumbs
- Polished the bottom navigation (next/back buttons)
- Wider min-width on content
image
image
image

## Changelog [New | Bug | Docs]

`NOCHANGELOG`

### CODE REVIEW

**Review by PedramNavid — APPROVED** [review]
so beautiful

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (PedramNavid) [inline yarn.lock:1]: was this meant to be checked in?:

**Comment by cmpadden** [comment]
Ohhhh heck yeah

**Review by erinkcochran87 — APPROVED** [review]
It's so beautiful 😭

**Pull request merged**